strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/antlr4/org/carlspring/strongbox/nuget/filter/NugetODataFilter.g4

```diff
@@ -94,7 +94,7 @@ NOT
 
 VALUE
 :
-    [a-zA-Z_0-9.]+
+    [a-zA-Z_0-9.\-]+
```

Comment:
@grantfar could the package name starts with `-`? if not then probably better to improve the expression.
Also it will be good if you can provide relevant test cases within `NugetFilterODataParserTestCase`